angular/projects/admin/src/app/admin/document-type/document-type-list/document-type-list.component.html

```diff
@@ -16,7 +16,7 @@ <h4 class="mat-h2 document-title">{{doc.title}}</h4>
             </div>
             <mat-divider></mat-divider>
           </li>
-          <mat-progress-bar *ngIf="!isLastItem"
+          <mat-progress-bar *ngIf="!isLoading"
```

Comment:
Remove the exclamation mark

Reply:
Resolved https://github.com/oddbit/tanam/pull/248/commits/92d754b713a724a5be3f6862b90d2f6778452af1